packages/flutter_tools/test/general.shard/base/process_test.dart

```diff
@@ -4,21 +4,15 @@
 
 // @dart = 2.8
 
-import 'dart:async';
-
-import 'package:fake_async/fake_async.dart';
 import 'package:flutter_tools/src/base/io.dart';
 import 'package:flutter_tools/src/base/logger.dart';
 import 'package:flutter_tools/src/base/platform.dart';
 import 'package:flutter_tools/src/base/process.dart';
 import 'package:flutter_tools/src/base/terminal.dart';
-import 'package:mockito/mockito.dart';
 
 import '../../src/common.dart';
 import '../../src/fake_process_manager.dart';
 import '../../src/fakes.dart';
-import '../../src/mocks.dart' show MockProcessManager,
-                                   flakyProcessFactory;
```

Comment:
`MockProcessManager`, `flakyProcessFactory`, and `_ProcessFactory` look dead on this PR, if you'd like the joy of deleting those as well.

In fact if you move `MockAndroidSdk` into `android_sdk_test` you can delete mocks.dart totally.

Reply:
Ahh thats right, I already landed the PR that removed that usage. Will update